### DISCUSSION TITLE
forum-session-Universal-Designs

### DISCUSSION THREAD

**Discussion by giscus** [arXiv/accessibility2024#132] [General]
# forum-session-Universal-Designs

There is no one right way to approach accessibility, but understanding different approaches can help us be much more accessible. Three brilliant thought leaders, authors, and powerful advocates share their understanding of accessibility.

https://accessibility2024.arxiv.org/forum-session-Universal-Designs

**Comment by SBBCornell** [comment]
Welcome everyone! Please share thoughts, questions, and resources of interest to the community.
